src/utility/hooks.js

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useState } from "react"
```

Comment:
Great use of hooks. If you had hooks with more logic and code, I would suggest making a file for each hook